cluster-scope/overlays/ocp-prod/kustomization.yaml

```diff
@@ -27,6 +27,8 @@ resources:
   - ../../bundles/ocs
   - ../../bundles/vertical-pod-autoscaler
 
+  - apiservers/cluster.yaml
+  - certificates/api-certificate-letsencrypt.yml
```

Comment:
do you want to change extension to `*.yaml` to keep things consistent?

Reply:
Whoops, yes. I absolutely do.